micah-akpan/bucket-list-api
# Chore: Setup Babel + TypeScript

#### What does this PR do?
This PR sets up Babel to enable development in ES6 + Integrate TypeScript Compiler
#### Description of Task to be completed?
- Add Dev tools to ease development (Babel and TypeScript Compiler)
#### How should this be manually tested?
- Git fetch and check out into chore/setup-typescript
- Install deps: `yarn`
- Start server: `yarn start`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168602838](https://www.pivotaltracker.com/story/show/168602838)
